### PR TITLE
Periodic process manager 

### PR DESCRIPTION
**Pull request by jedrz**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -286,6 +286,7 @@ lazy val dist = {
         (crossTarget in generic).value / "genericModel.jar" -> "model/genericModel.jar",
         (crossTarget in demo).value / s"demoModel.jar" -> "model/demoModel.jar",
         (crossTarget in flinkProcessManager).value / s"nussknacker-flink-manager.jar" -> "managers/nussknacker-flink-manager.jar",
+        (crossTarget in flinkPeriodicProcessManager).value / s"nussknacker-flink-periodic-manager.jar" -> "managers/nussknacker-flink-periodic-manager.jar",
```
Comment (mproch) [inline build.sbt:289]: I think it's not needed ATM, as periodic manager currently can be used only via custom extension (we don't expose any ProcessManagerProvider via ServiceLoader)

File: build.sbt [hunk 2]
```diff
@@ -936,4 +956,4 @@ lazy val root = (project in file("."))
   )
 
 addCommandAlias("assemblySamples", ";flinkManagementSample/assembly;standaloneSample/assembly;demo/assembly;generic/assembly")
-addCommandAlias("assemblyEngines", ";flinkProcessManager/assembly;engineStandalone/assembly")
+addCommandAlias("assemblyEngines", ";flinkProcessManager/assembly;flinkPeriodicProcessManager/assembly;engineStandalone/assembly")
```
Comment (mproch) [inline build.sbt:959]: I think it's not needed ATM, as periodic manager currently can be used only via custom extension (we don't expose any ProcessManagerProvider via ServiceLoader)

File: ui/runServer.sh [hunk 3]
```diff
@@ -10,9 +10,10 @@ SCALA_VERSION=${SCALA_VERSION:-2.12}
 PROJECT_BASE_DIR="../../.."
 #management jars are currently needed to access ProcessManagers
 FLINK_ENGINE_JAR=$PROJECT_BASE_DIR/engine/flink/management/target/scala-${SCALA_VERSION}/nussknacker-flink-manager.jar
+FLINK_PERIODIC_ENGINE_JAR=$PROJECT_BASE_DIR/engine/flink/management/target/scala-${SCALA_VERSION}/nussknacker-flink-periodic-manager.jar
 STANDALONE_ENGINE_JAR=$PROJECT_BASE_DIR/engine/standalone/engine/target/scala-${SCALA_VERSION}/nussknacker-standalone-manager.jar
 
-export CLASSPATH="../target/scala-${SCALA_VERSION}/nussknacker-ui-assembly.jar:$FLINK_ENGINE_JAR:$STANDALONE_ENGINE_JAR"
+export CLASSPATH="../target/scala-${SCALA_VERSION}/nussknacker-ui-assembly.jar:$FLINK_ENGINE_JAR:$FLINK_PERIODIC_ENGINE_JAR:$STANDALONE_ENGINE_JAR"
```
Comment (mproch) [inline ui/runServer.sh:16]: I think it's not needed ATM, as periodic manager currently can be used only via custom extension (we don't expose any ProcessManagerProvider via ServiceLoader)

**Review by mproch — COMMENTED** [review]
Let's keep it in preview for a moment ;)

**Review by nadberezny — APPROVED** [review]

**Pull request merged**